app/views/prison_mailer/booked.html.erb

```diff
@@ -5,35 +5,35 @@
 <table cellspacing="10">
   <tr>
     <th valign="top" align="left">Slot granted</th>
-    <td><%= format_slot_for_public(@visit.slot_granted) %></td>
+    <td><%= format_slot_for_public(@booking_response.slot_granted) %></td>
   </tr>
   <tr>
     <th valign="top" align="left">Prisoner name</th>
-    <td><%= @visit.prisoner_full_name %></td>
+    <td><%= @booking_response.prisoner_full_name %></td>
   </tr>
   <tr>
     <th valign="top" align="left">Prisoner number</th>
-    <td><%= @visit.prisoner_number %></td>
+    <td><%= @booking_response.prisoner_number %></td>
   </tr>
   <tr>
     <th valign="top" align="left">Approved visitors</th>
     <td>
-      <% @visit.visitors.allowed.each do |visitor| %>
+      <% @booking_response.visitors.allowed.each do |visitor| %>
```

Comment:
We still need to update this to use the in memory data from the booking response instead of loading the allowed visitors from the database. In a preview the allowed visitors from the database will always be all the visitors